go/vt/vtgate/planbuilder/testdata/aggr_cases.json

```diff
@@ -3249,72 +3242,6 @@
       ]
     }
   },
-  {
-    "comment": "select (select 1 from user u having count(ue.col) > 10) from user_extra ue",
-    "query": "select (select 1 from user u having count(ue.col) > 10) from user_extra ue",
-    "plan": {
-      "QueryType": "SELECT",
-      "Original": "select (select 1 from user u having count(ue.col) > 10) from user_extra ue",
-      "Instructions": {
-        "OperatorType": "Subquery",
-        "Variant": "PulloutValue",
-        "PulloutVars": [
-          "__sq1"
-        ],
-        "Inputs": [
-          {
-            "InputName": "SubQuery",
-            "OperatorType": "SimpleProjection",
-            "Columns": [
-              0
-            ],
-            "Inputs": [
-              {
-                "OperatorType": "Filter",
-                "Predicate": ":1 > 10",
-                "Inputs": [
-                  {
-                    "OperatorType": "Aggregate",
-                    "Variant": "Scalar",
-                    "Aggregates": "any_value(0) AS 1, sum_count(1) AS count(ue.col)",
-                    "Inputs": [
-                      {
-                        "OperatorType": "Route",
-                        "Variant": "Scatter",
-                        "Keyspace": {
-                          "Name": "user",
-                          "Sharded": true
-                        },
-                        "FieldQuery": "select 1, count(ue.col) from `user` as u where 1 != 1",
-                        "Query": "select 1, count(ue.col) from `user` as u",
```

Comment:
don't know how this got through, but this query is clearly not functional. We are trying to read `ue.col`, but there is no `ue` in the `FROM` clause here.

Reply:
This looks like it is coming because of a derived table. Possibly fixed by what we changed today?